### ISSUE TITLE
why you use SSE loss in classification model?

### ISSUE THREAD

**Issue by gaopeng-eugene** [pjreddie/darknet#212]

**Comment by gaopeng-eugene** [comment]
I checked the darknet.cfg and imagenet.cfg. It seems that you use SSE loss after softmax activation.

**Comment by gaopeng-eugene** [comment]
Anyone notice this?

**Comment by pjreddie** [comment]
shhhhhh.....

**Comment by pjreddie** [comment]
it's really cross entropy something something which is just SSE as long as you don't do anything backward in the softmax layer which I don't because i don't understand how to take the gradient of a softmax and who ever uses softmax for anything that isn't classification? so anyway, it'll work the way you want it to work.

**Comment by TaihuLight** [comment]
@pjreddie @gaopeng-eugene 
https://github.com/pjreddie/darknet/pull/96/commits
fixes the backward of the softmax layer, I suggest to add it to this repo.
